modules/custom/az_paragraphs/az_paragraphs_text_media/js/az_paragraphs_az_text_media_vimeo.es6.js

```diff
@@ -0,0 +1,172 @@
+(($, Drupal, once) => {
+  Drupal.behaviors.az_vimeo_video_bg = {
+    attach(context, settings) {
+      // Error messaging function
+      function vimeoError(error) {
+        switch (error.name) {
+          case 'PasswordError':
+            console.log('The Vimeo video is password-protected.');
+            break;
+          case 'PrivacyError':
+            console.log('The Vimeo video is private.');
+            break;
+          default:
+            console.log(
+              `Some errors occurred with the Vimeo video: ${error.name}`,
+            );
+            break;
+        }
+      }
+
+      if (window.screen && window.screen.width > 768) {
+        // @see https://developer.vimeo.com/player/sdk/basics
+        // Defaults
+        const defaults = {
+          vimeoId: '',
+          width: $(window).width(),
+          ratio: 16 / 9,
+          autoplay: true,
```

Comment:
This change, along with the one below, should fix the issue of only one video playing when multiple Vimeo videos are on a page as Text on Media backgrounds. (The .js file will need to be rebuilt after making these changes to the .es6.js file.)

```suggestion
          autopause: false,
          autoplay: true,
```